components/post_view/post_info/post_info.jsx

```diff
@@ -192,14 +199,19 @@ export default class PostInfo extends React.PureComponent {
                             onEmojiClick={this.reactEmojiClick}
                             rightOffset={7}
                         />
-                        <button
-                            className='reacticon__container color--link style--none'
-                            onClick={this.toggleEmojiPicker}
+                        <ChannelPermissionGate
+                            channelId={post.channel_id}
+                            teamId={this.props.teamId}
+                            perms={[Permissions.ADD_REACTION]}
```

Comment:
`perms` don't sound natural to me. :)

Reply:
Yes, actually we have changed in the past the `ChannelPermGate` to `ChannelPermissionGate` we can use the entire word everywhere.

Reply:
I'll do it in another commit renaming all perms to permissions.